frontend/src/features/commonComponents/Modals/Delete.tsx

```diff
@@ -1,29 +1,40 @@
 import { Accent, Button, Dialog } from '@mtes-mct/monitor-ui'
 
 type DeleteModalProps = {
+  cancelButtonText?: string
   context: string
   isAbsolute?: boolean
   onCancel: () => void
   onConfirm: () => void
   open: boolean
   subTitle: string
+  title: string
 }
 
-export function DeleteModal({ context, isAbsolute = true, onCancel, onConfirm, open, subTitle }: DeleteModalProps) {
+export function DeleteModal({
+  cancelButtonText,
+  context,
+  isAbsolute = true,
+  onCancel,
+  onConfirm,
+  open,
+  subTitle,
+  title
+}: DeleteModalProps) {
   if (!open) {
     return null
   }
 
   return (
     <Dialog isAbsolute={isAbsolute}>
-      <Dialog.Title>Enregistrer les modifications ?</Dialog.Title>
+      <Dialog.Title>{title}</Dialog.Title>
       <Dialog.Body>
         <p>{subTitle}</p>
       </Dialog.Body>
 
       <Dialog.Action>
         <Button accent={Accent.SECONDARY} name={`delete-${context}-modal-cancel`} onClick={onCancel}>
-          Retourner à l&apos;édition
+          {cancelButtonText ?? 'Retourner à l&apos;édition'}
```

Comment:
C'est un micro-détail que j'ai noté sur Env, on peut passez un "vrai apostrophe" en JSX : `’`.